spec/factories/course_experience_points_records.rb

```diff
@@ -11,10 +11,15 @@
         build(:course_user, course: course, user: user)
     end
     points_awarded { rand(1..20) * 100 }
-    reason 'EXP for some event'
+    actable_type 'self::PointsAwardingDummy'
```

Comment:
A dummy model defined in the factory would be good.

In spec/factories/course_experience_points_records.rb<https://github.com/Coursemology/coursemology2/pull/909#discussion_r55920185>:

> @@ -11,10 +11,15 @@
>          build(:course_user, course: course, user: user)
>      end
>      points_awarded { rand(1..20) * 100 }
> -    reason 'EXP for some event'
> +    actable_type 'self::PointsAwardingDummy'


Okay, so I should create a dummy model with its database table?

—
Reply to this email directly or view it on GitHub<https://github.com/Coursemology/coursemology2/pull/909/files#r55920185>.